### DISCUSSION TITLE
For me a new window when starting CoursePlay

### DISCUSSION THREAD

**Discussion by Toaogurch** [Courseplay/Courseplay_FS22#3303] [General]
First of all, I have to tell you that I have Alzheimer's disease.
This only means that I have memory gaps!

Since I only need English from time to time, I have to rely on the help of Deepl.

I haven't played LS22 for several months now. Now I want to play again but I can't cope with the new monitor from Courseplay!

Is there a video on how to use Courseplay now?

Thank you very much for your help and effort in advance!

Georg Offermann

**Comment by Tensuko** [comment]
Only by Youtuber and ingame help menu.
New window? Best is you make a screenshot of the new window to explain to you.

**Comment by Toaogurch** [comment]
Hallo Tensuko, ich schreibe lieber in deutsch weil ich englisch nicht richtig beherrsche!
Ich hoffe das ich das Foto angehängt habe. So habe ich noch nicht Courseplay bedient. Wo finde ich die Beschreibung um auch zwei Maschinen das gleiche Feld gleichzeitig bearbeiten zu lassen? Wo finde ich die Beschreibung wie ich jetzt das wunderbare Courseplay bearbeiten zu können?

Herzlichen Dank im voraus für Deine Hilfe!!!

Liebe Grüße!   
Gurch

**Reply by Tensuko** [reply]
Du kannst z.B. auf courseplay.dev alles nachlesen (musst du nur auf deutsch umschalten), oder im Spiel haben wir die Hilfe auch.
Wenn du dann noch etwas nicht verstehst, kannst du gerne nach fragen.

Bild ist leider nicht zu sehen. Einfach mit der Maus in das Fenster ziehen dann wird es hochgeladen.

**Comment by Toaogurch** [comment]
Hallo Tensuko,
ich habe eine Frage, nach dem neuen Update kann ich keine Feldarbeit von mehreren Helfen erledigen! Ist das gewollt oder habe ich etwas übersehen? Ich werden nicht mehr gefragt mit wievielten Helfen ich arbeiten möchte!!
Ich finde Courseplay einfach Phenomenal!!!
Herzlichen Dank für Deine Mühe im Voraus!  

Liebe Grüße!

**Reply by Tensuko** [reply]
Eigentlich müsste ich jetzt sagen ingame hilfe lesen....
Schalte mal den Experten Modus ein in den globalen Einstellungen.

**Comment by Toaogurch** [comment]
Hallo Tensuko,

ich muss Dich eigentlich auf mehrere Bier 🍻 einladen!

Es könnte vieles so einfach sein, wenn man auch lesen würde!!!!

Ganz herzlichen Dank und entschuldige meinen Fehler!!!!

Ihr macht eine super Arbeit!!!!

Georg Offermann
Hardenbergstraße. 68
45472Mülheim an der Ruhr
TEL.:01634759059
Mail.: ***@***.***
Am 18. Okt. 2024, 16:35 +0200 schrieb Tensuko ***@***.***>:

**Reply by Tensuko** [reply]
Ach du alles gut :)
Wenns nun klappt, freut es mich, weniger Arbeit für uns :D
